tools/benchmark/memory/src/main.rs

```diff
@@ -200,7 +200,7 @@ fn main() {
                 );
                 process::exit(1);
             }
-            (*parts.get(0).unwrap(), *parts.get(1).unwrap())
+            (*parts.first().unwrap(), *parts.get(1).unwrap())
```

Comment:
Another weakly held inappropriate `get(0)`

Reply:
another candidate for refactor.